src/main/java/org/edumips64/utils/cli/ConfigCommand.java

```diff
@@ -0,0 +1,39 @@
+package org.edumips64.utils.cli;
+
+import org.edumips64.utils.ConfigKey;
+import org.edumips64.utils.ConfigStore;
+import org.edumips64.utils.CurrentLocale;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.ParentCommand;
+
+import java.util.stream.Stream;
+
+@Command(name="config")
+public class ConfigCommand implements Runnable {
+
+    @ParentCommand
+    private Cli cli;
+
+    private static ConfigStore configStore;
+
+    @Override
+    public void run() {
+        configStore = cli.getConfigStore();
+        System.out.println(CurrentLocale.getString("CLI.CONFIG.HEADER"));
+        Stream.of(ConfigKey.values()).forEach(ConfigCommand::printValue);
+    }
+
+    private static void printValue(ConfigKey k) {
```

Comment:
Nice way to print the value not knowing the type! :) I think `ConfigStore` could have a way to make this easier (for example a `printValue(ConfigKey key)` method, but since this is the only use case I can foresee for such a method I think that the workaround you have here is absolutely fine!

Reply:
Thanks, I looked for a convenience method there but didn't see it. That would probably be a good add. Of course, I could be looking in the wrong spot